opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -109,20 +109,27 @@ message Span {
   // field must be empty. The ID is an 8-byte array.
   bytes parent_span_id = 4;
 
-  // Flags, a bit field. 8 least significant bits are the trace
-  // flags as defined in W3C Trace Context specification. Readers
-  // MUST not assume that 24 most significant bits will be zero.
+  // Flags, a bit field. Bits 0-7 (8 least significant bits) are the trace
+  // flags as defined in W3C Trace Context specification.
   // To read the 8-bit W3C trace flag, use `flags & SPAN_FLAGS_TRACE_FLAGS_MASK`.
   //
   // When creating span messages, if the message is logically forwarded from another source
   // with an equivalent flags fields (i.e., usually another OTLP span message), the field SHOULD
   // be copied as-is. If creating from a source that does not have an equivalent flags field
-  // (such as a runtime representation of an OpenTelemetry span), the high 24 bits MUST
+  // (such as a runtime representation of an OpenTelemetry span), the high 22 bits MUST
   // be set to zero.
   //
   // [Optional].
```

Comment:
@tigrannajaryan, does this `[Optional]` reference refer to the flags overall or the trace flag?

Reply:
cc @Oberon00

Reply:
I just moved this around, I thought it referred to the field as a whole.